src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt

```diff
@@ -42,7 +42,8 @@ open class CodyIntegrationTextFixture : BasePlatformTestCase() {
 
   override fun tearDown() {
     try {
-      FixupService.getInstance(myFixture.project).getActiveSession()?.apply {
```

Comment:
Shouldn't that whole file be removed and that line modified in the `CodyIntegrationTextFixture.kt`?
Then git should correctly show it as rename, and not like now as whole new file added.

Reply:
Good catch; fixed that.